docs/source/rpc.rst

```diff
@@ -9,6 +9,9 @@ training through a set of primitives to allow for remote communication, and a
 higher-level API to automatically differentiate models split across several
 machines.
 
+.. warning::
+  The RPC API is experimental and subject to change.
```

Comment:
@mrshenli Can we expand this to also say the API is in "beta" mode in the sense that this is not a stable release and there might be bugs/edge cases that haven't been addressed yet. Or do we usually use "experimental" to denote both that the API might change and also that the user shouldn't expect this to be a stable release.

Reply:
In other doc pages, it looks like just this experimental warning appears.